docs/source/en/using-diffusers/controlnet.md

```diff
@@ -0,0 +1,449 @@
+# ControlNet
+
+ControlNet is a type of model for controlling image diffusion models by conditioning the model with an additional input image. There are many types of conditioning inputs (canny edge, user sketching, human pose, depth, and more) to control a diffusion model. This is hugely useful because it affords you greater control over image generation, making it easier to generate specific images without fiddling with text prompts or denoising values as much.
+
+<Tip>
+
+Check out Section 3.5 of the [ControlNet](https://huggingface.co/papers/2302.05543) paper for a list of ControlNet implementations of various conditioning inputs. You can find all available ControlNet conditioned models on [lllyasviel](https://huggingface.co/lllyasviel)'s Hub profile.
```

Comment:
Users won't be able to find any ControlNet checkpoints for SDXL under their profile, though.

Reply:
I see. You have linked hf.co/diffusers at the bottom for the SDXL ones. 

Maybe we could also encourage the users to check out the community-trained ControlNets too? 

* SDXL: https://huggingface.co/models?other=stable-diffusion-xl&other=controlnet
* SD: https://huggingface.co/models?other=stable-diffusion&other=controlnet